jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyConfigurationUtilsTest.java

```diff
@@ -27,31 +27,29 @@
 import org.eclipse.jkube.kit.common.AssemblyFileSet;
 import org.eclipse.jkube.kit.config.image.build.Arguments;
 import org.eclipse.jkube.kit.config.image.build.BuildConfiguration;
-
-import mockit.Expectations;
-import mockit.Injectable;
 import org.junit.Test;
-
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.createDockerFileBuilder;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getAssemblyConfigurationOrCreateDefault;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getJKubeAssemblyFileSets;
 import static org.eclipse.jkube.kit.build.api.assembly.AssemblyConfigurationUtils.getJKubeAssemblyFiles;
 
+
 public class AssemblyConfigurationUtilsTest {
 
+  private BuildConfiguration mockedBuildConfiguration;
+  private Assembly assembly;
+  AssemblyFile file;
```

Comment:
In addition to Rohan's comments:
- https://github.com/eclipse/jkube/pull/1617/files#r923217234
- https://github.com/eclipse/jkube/pull/1617/files#r923217589

If these variables are initialized in a test method/case, then use local variables.

If they need to be instance variables, then set them up in a common setUp method.

Reply:
As these were set up locally, have used local variables here ,Thanks!